examples/k8s/deploy/linstor-csi-1.17.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       hostNetwork: true
       containers:
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.4.0
+          image: quay.io/k8scsi/csi-provisioner:v0.5.0
```

Comment:
Hi, thank you for the PR.  Surely you meant v1.5.0 here, though?

Reply:
My bad, fixed!